### ISSUE TITLE
Every 10 minutes server upload spike

### ISSUE THREAD

**Issue by crazypete79** [StrangeLoopGames/EcoIssues#14948]
I'm running a server at home. Version 0.8.3.3
Every 10 minutes the upload speed of the server spikes.
The other values are pretty much constant at low values.
Disk, CPU and memory usage are also constant and in normal range.

I found similar issues, but those were about CPU spikes. Though they also happened every 10 minutes, so they could be related.

I have tried to change the UPnP setting I found recommended but that didn't help.
Also changing the auto-save and backup interval didn't help.
I did notice that the fan in my PC slows down when it happens.

I have also tried to run it on another computer, but the same issue remains.
For players the result is lag and when it hits the cap of my ISP even disconnects.

The other 9 minutes it runs fine. It's just that spike in the 10th minute.

I've added screens of the network tab and of my resource manager.
These were taken from my work PC where I have it running now, but it's basically the same on the dedicated PC. The dedicated PC runs the server from SSD, my work PC runs it from my NAS.

The timeline is with the UPnP setting disabled, when enabled the time it takes to go back to normal is shorter, but the peak is higher.

Hope this is enough info and that there is a solution or workaround for this.

![Normal](https://user-images.githubusercontent.com/43097909/75635996-1b4b1800-5c1b-11ea-91b7-1faa624965d0.jpg)
![Spike](https://user-images.githubusercontent.com/43097909/75635997-1be3ae80-5c1b-11ea-9dbf-e7954c5a661c.jpg)
![Spike time](https://user-images.githubusercontent.com/43097909/75635998-1be3ae80-5c1b-11ea-9659-292a1b70ca87.jpg)

**Comment by crazypete79** [comment]
Just a quick update for you.
Thanks to one of the players on my server I moved the whole server directory to a real server with a much better internet connection speed.
After that the server ran fine. I couldn't find any weird upload spikes anymore.
The only real difference was that it runs on windows server edition and the much better internet speed. 
Although I did notice that the upload speed per player seemed to change more. On my crappy speed it was usually less then 10 k/s, on the speedy server there were more times it jumped to 15 to 30 k/s for a few seconds.
I hope it helpes.
